Added slideshow of school talk testimonials

CharlzGaldo opened the pull request:
It Should be working fine, let me know if you find anything wrong or needs changing. I had to change some "class" names in the code for the slideshow like `fade` -> `fades` and `active` -> `actives` because it was already used in previous code and changed code that was already written.

dhood commented:
@CharlzGaldo this is awesome! looks incredible, and it's something that I have never done myself and now can learn from you :sunglasses: how cool.

Can you make it smaller, please? maybe about 70% of its current size? and smaller on mobile, because it currently is disproportionately large: 
![image](https://github.com/dhood/dhood.github.io/assets/5618076/43f5455a-dd79-44a1-9517-b03aed8ac6fd)

CharlzGaldo commented:
My browser doesn't show the mobile device view properly so please LMK if the problem is fixed.

dhood commented:
it is fixed! although the font size of the description text is a bit too big:
![Screenshot from 2024-02-13 10-43-56](https://github.com/dhood/dhood.github.io/assets/5618076/75ee06b7-2b7e-4e5f-af46-4e61aef1a383)

Can you please remove the description text, altogether? can it be left blank?
(ps to get it to switch to mobile view I just make my browser window skinnier and once the width is less than a certain amount it switches modes :+1: )

You have encountered a new challenge accidetnally which is a "merge conflict". Have a google and try to fix it! if anything goes wrong, I will be able to restore the current state of the branch from my local version. LMK where you get stuck :slightly_smiling_face:

CharlzGaldo commented:
Nice! I found a way to make it go to mobile view on my end now. I removed the text but I don't know if I fixed the merge conflict correctly 😅 Hopefully I didn't break anything.

dhood commented on index.html:
```diff
@@ -935,8 +937,9 @@ <h3>2006</h3>
-				<a href="#me" class="icon fa-home" style="text-decoration:none"><span>Home</span></a>
-				<a href="#work" class="icon fa-folder active" style="text-decoration:none"><span>Work</span></a>
+
+				<a href="#me" class="icon fa-home active" style="text-decoration:none"><span>Home</span></a>
```
I think these might not be correct anymore! did you mean to change them all? can you explain your goal? From memory they were showing as "headers" on mobile version for what "page" the person is on/where they're up to

dhood replied:
nvm it was easy to fix myself! since I want to see this beautiful slideshow on the website because I need to show it to someone :grin: thanks again for the feature!!!

CharlzGaldo replied:
Sorry! Yes they are supposed to be active on the correct tabs, I made the mistake thinking they were all supposed to start at "home"" so I gave it all the "active" when the website started, but It makes more sense that they are active for the specific tabs they are supposed to be.

dhood commented:
nice work fixing the merge conflict.. that is a very useful skill. hopefully we get to practice it some more!

CharlzGaldo commented:
OH! I see what I did wrong there, sorry, I thought that all those tabs needed to be active on the "HOME" but it makes more sense to be active depending on the tab they are on. Sorry for the mistake.

dhood commented:
not a problem, it wasn't obvious!! :grin:
